outputs.tf

```
@@ -11,3 +11,21 @@ output "iam-policy-document-athena-workspace-use" {
 
   value = data.aws_iam_policy_document.iam-policy-document-athena-workspace-use.json
 }
+
+output "athena_workgroup" {
+  description = "Athena workgroup."
+
+  value = aws_athena_workgroup.this
+}
+
+output "bucket" {
```

Let's keep it symmetric with the other outputs:

```suggestion
output "s3_bucket" {
```